go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/miguelmatosduarte/instrumentedsql
```

Comment:
Shouldn't this be module `github.com/pcruz7/instrumentedsql`?

Reply:
it should, fixed :)